src/v2/stack_aggregator.py

```diff
@@ -26,9 +27,11 @@
 from src.v2.normalized_packages import NormalizedPackages, GoNormalizedPackages
 from src.v2.license_service import (get_license_analysis_for_stack,
                                     get_license_service_request_payload)
+from collections import namedtuple
 
 logger = logging.getLogger(__name__)
 _TRUE = ['true', True, 1, '1']
+_PACKAGE = namedtuple("Package", ["package", "version"])
```

Comment:
Just noticed, defining an arbitrary type from consumer is not a right model. It should come from the utils. It should have caught during https://github.com/fabric8-analytics/fabric8-analytics-utils/pull/83/files, but slipped.

I would recommend to fix that first by introducing a dataclass or normal class or named tuple into the utils.